snippets/csharp/System/NullReferenceException/Overview/Chain1.cs

```diff
@@ -1,62 +1,71 @@
-﻿// <Snippet6>
-using System;
+﻿using System;
 
-public class Example
+namespace Chain1
 {
-   public static void Main()
-   {
-      var pages = new Pages();
-      if (!String.IsNullOrEmpty(pages.CurrentPage.Title)) {
-         String title = pages.CurrentPage.Title;
-         Console.WriteLine("Current title: '{0}'", title);
-      }
-   }
-}
+    // <Snippet6>
+    public class Chain1Example
+    {
+        public static void Main()
+        {
+            var pages = new Pages();
+            if (!string.IsNullOrEmpty(pages.CurrentPage.Title))
+            {
+                string title = pages.CurrentPage.Title;
+                Console.WriteLine("Current title: '{0}'", title);
```

Comment:
string interpolation:

```suggestion
                Console.WriteLine($"Current title: '{title}'");
```